Remotion/Validation/UnitTests/Attributes/Validation/LengthValidationAttributeTest.cs

```diff
@@ -29,33 +29,48 @@ namespace Remotion.Validation.UnitTests.Attributes.Validation
   [TestFixture]
   public class LengthValidationAttributeTest
   {
-    private LengthValidationAttribute _attribute;
+    private LengthValidationAttribute _lengthValidationAttribute;
+    private LengthValidationAttribute _exactLengthValidationAttribute;
+    private LengthValidationAttribute _minimumLengthValidationAttribute;
+    private LengthValidationAttribute _maximumLengthValidationAttribute;
     private Mock<IValidationMessageFactory> _validationMessageFactoryStub;
 
     [SetUp]
     public void SetUp ()
     {
-      _attribute = new LengthValidationAttribute(10, 20);
+      _lengthValidationAttribute = new LengthValidationAttribute() { MinLength = 10, MaxLength = 20 };
+      _exactLengthValidationAttribute = new LengthValidationAttribute() { MinLength = 10, MaxLength = 10 };
+      _minimumLengthValidationAttribute = new LengthValidationAttribute() { MinLength = 10 };
+      _maximumLengthValidationAttribute = new LengthValidationAttribute() { MaxLength = 10 };
```

Comment:
Best practice: originally we had the same setup for each test. it is best practice to keep individual setup in the test itself. In particular, here we create different attributes, each is used just once. This should be in a local variable in the individual tests. An added advantage of this is that you ensure you are using the correct value in the test and do not accidentally access use the wrong field. (yes, this can happen. quite easily, too, with copy&paste :) )